package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "joliGEN",
+    "description": "Generative AI Toolset with GANs and Diffusion for Real-World Applications",
+    "version": "5.0.0",
```

Comment:
version 5.0.0?

Reply:
Fixed, thanks, leaked from the script trials.